.circleci/config.yml

```
@@ -34,4 +34,6 @@ jobs:
               exit 0;
             fi
 
+            echo "run npm build and commit the main.bundle.js";
```

Small change to the wording but not critical:

```suggestion
            echo -e "Your build failed as the latest changes were not included in the built JS files";
            echo -e "To fix this, run npm build and commit the main.bundle.js changes";
```